test/ctia/test_helpers/access_control.clj

```diff
@@ -951,86 +951,110 @@
                                                   (:parsed-body player-3-entity-repost2)]})))))))
 
 (defn test-access-control-tlp-settings
-  [entity new-entity]
-  (testing "TLP Settings Enforcement"
-    (swap! (p/global-properties-atom) assoc-in
-           [:ctia :access-control]
-           {:default-tlp "amber"
-            :min-tlp "amber"})
-
-    (let [{status-default-tlp :status
-           body-default-tlp :parsed-body}
-          (post (format "ctia/%s" entity)
-                :body (dissoc new-entity :tlp)
-                :headers {"Authorization" "player-1-token"})
-          {status-disallowed-tlp :status
-           body-disallowed-tlp :parsed-body}
-          (post (format "ctia/%s" entity)
-                :body (assoc new-entity :tlp "white")
-                :headers {"Authorization" "player-1-token"})]
-
-      (is (= 201 status-default-tlp))
-      (is (= "amber" (:tlp body-default-tlp)))
-
-
-      (is (= 400 status-disallowed-tlp))
-      (is (= "Invalid document TLP white, allowed TLPs are: amber,red"
-             (:message body-disallowed-tlp))))))
-
+  [entity new-entity fixtures]
+  (helpers/with-config-transformer*
+    #(assoc-in % [:ctia :access-control] {:default-tlp "amber"
+                                          :min-tlp "amber"})
+    #(fixtures
+       (fn []
+         (testing "TLP Settings Enforcement"
+           (let [;; verify the with-config-transformer* call above--it's possible
+                 ;; a fixture could override it since we call it first
+                 get-in-config (helpers/current-get-in-config-fn)
+                 _ (assert (= (get-in-config [:ctia :access-control])
+                              {:default-tlp "amber"
+                               :min-tlp "amber"})
+                           (get-in-config [:ctia :access-control]))
```

Comment:
There is an unfortunate ordering issue here. Ideally, we'd call `with-config-transformer*` inside the function we pass to `fixtures`, but `fixtures` starts CTIA, so it's "too late" to do that (for reasons described in PR description).

To workaround this, we double check our `with-config-transformer*` call has not been overwritten by another fixture via this assertion.